apps/src/applab/appStorage.js

```diff
@@ -233,15 +232,14 @@ var handleDeleteRecord = function(tableName, record, onSuccess, onError) {
     return;
   }
   if (this.status === 404) {
-    onError('error deleting record: could not find record id ' + record.id +
-        ' in table ' + tableName);
+    onSuccess(false);
     return;
   }
   if (this.status < 200 || this.status >= 300) {
     onError('error deleting record: unexpected http status ' + this.status);
     return;
   }
-  onSuccess();
+  onSuccess(true);
```

Comment:
Yes, it probably should be. Do you think it makes sense to rename this at any of these layers in the code now?
```
api.js: exports.deleteRecord = function (table, record, onSuccess, onError) {
applabCommands.deleteRecord = function (opts) { // opts.onSuccess
AppStorage.deleteRecord = function(tableName, record, onComplete, onError) {
var handleDeleteRecord = function(tableName, record, onComplete, onError) {
```